sdks/python/container/run_validatescontainer.sh

```diff
@@ -47,9 +55,15 @@ gcloud -v
 
 # Build the container
 TAG=$(date +%Y%m%d-%H%M%S)
-CONTAINER=us.gcr.io/$PROJECT/$USER/python
-echo "Using container $CONTAINER"
-./gradlew :beam-sdks-python-container:docker -Pdocker-repository-root=us.gcr.io/$PROJECT/$USER -Pdocker-tag=$TAG --info
+if [[ -z $1 ]]; then
```

Comment:
I'd suggest to require explicit parameter both for py2 and py3, and have once condition where based on the parameter we set:
 ```
IMAGE_NAME="python3"  # Use this to create CONTAINER_IMAGE variable.
PYTHON_INTERPRETER="python3.5"   # Use this in virtualenv command.
```

Reply:
sg.